api/src/wrangler.rs

```diff
@@ -13,7 +13,7 @@ use fvm_shared::{ActorID, MethodNum, BLOCK_GAS_LIMIT};
 use crate::{ActorState, Bench, ExecutionResult};
 
 pub struct ExecutionWrangler {
-    bench: RefCell<Box<dyn Bench>>,
+    pub bench: RefCell<Box<dyn Bench>>,
```

Comment:
Please proxy the `store()` method, and probably also return the BlockstoreWrapper from it. This is supposed to be the convenient API.